content/fr/docs/reference/glossary/application-architect.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Application Architect
```

Comment:
As this is a concept (and not really Kubernetes-specific or tied to code), I would be tempted to translate the title of this glossary entry. Anyone specifically searching the French localization for the English term will be able to find it even with the French spelling of architecte.

@ryanzidago what do you think?

Reply:
@sftim alright.
 
I translated the title section in French too.